federation/ldap/pom.xml

```diff
@@ -29,6 +29,12 @@
     <name>Keycloak LDAP UserStoreProvider</name>
     <description />
 
+    <properties>
+        <maven.compiler.release>11</maven.compiler.release>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+    </properties>
```

Comment:
This allows usage of `Map.of(...` method. AFAIK jdk8 support was dropped some time ago already. 

Can we do this change?

Reply:
I think it is ok to have this here.